packages/manager/src/features/CloudPulse/Alerts/AlertsListing/AlertListing.tsx

```diff
@@ -1,88 +1,228 @@
+        return (
+          serviceFilters.some(
+            (serviceFilter) => serviceFilter.value === alert.service_type
+          ) &&
+          statusFilters.some(
+            (statusFilter) => statusFilter.value === alert.status
+          )
+        );
+      });
+    }
+
+    if (searchText) {
+      filteredAlerts = filteredAlerts.filter((alert: Alert) => {
+        return alert.label.toLowerCase().includes(searchText.toLowerCase());
+      });
+    }
+
+    return filteredAlerts;
+  }, [
+    alerts,
+    searchText,
+    serviceFilteredAlerts,
+    serviceFilters,
+    statusFilteredAlerts,
+    statusFilters,
+  ]);
+
+  if (alerts && alerts.length == 0) {
     return (
-      <Grid item xs={12}>
-        <Paper>
-          <StyledPlaceholder
-            subtitle="Start Monitoring your resources."
-            title=""
-          />
-        </Paper>
-      </Grid>
+      <StyledPlaceholder
+        buttonProps={[
+          {
+            children: 'Create Alerts',
+            onClick: () => {
+              history.push(`${url}/create`);
+            },
+          },
+        ]}
+        icon={AlertsIcon}
+        subtitle="Start monitoring your resources."
+        title=""
+      />
     );
   }
+
   return (
-    <Grid marginTop={2}>
-      <Table colCount={7} size="small">
-        <TableHead>
-          <TableRow>
-            {AlertListingTableLabelMap.map((value) => (
-              <TableSortCell
-                active={true}
-                direction={order}
-                handleClick={handleOrderChange}
-                key={value.label}
-                label={value.label}
-              >
-                {value.colName}
-              </TableSortCell>
-            ))}
-            <TableCell actionCell />
-          </TableRow>
-        </TableHead>
-        <TableBody>
-          {isError && (
-            <TableRowError
-              colSpan={7}
-              message={'Error in fetching the alerts.'}
-            />
-          )}
-          {isLoading && <TableRowLoading columns={7} />}
-          {alerts?.map((alert) => (
-            <AlertTableRow
-              handlers={{
-                handleDetails: () => handleDetails(alert),
-              }}
-              alert={alert}
-              key={alert.id}
-            />
-          ))}
-        </TableBody>
-      </Table>
-    </Grid>
+    <Stack spacing={2}>
+      <Box
+        alignItems={{ lg: 'flex-end', md: 'flex-start' }}
+        display="flex"
+        flexDirection={{ lg: 'row', md: 'column', sm: 'column', xs: 'column' }}
+        gap={3}
+        justifyContent="space-between"
+      >
+        <Box
+          flexDirection={{
+            lg: 'row',
+            md: 'column',
+            sm: 'column',
+            xs: 'column',
+          }}
+          display="flex"
+          gap={2}
+        >
+          <DebouncedSearchTextField
+            sx={{
+              width: { md: '300px', sm: '500px', xs: '300px' },
```

Comment:
This gets used 3 times throughout the file. How about storing it in a variable outside the component definition? Something like

```
  const searchAndSelectSx = {
    md: '300px',
    sm: '500px',
    xs: '300px',
  };
```

below the imports, and then use it like

```suggestion
              width: searchAndSelectSx,
```